### PR TITLE
AUTH-26: Go 1.16 for cluster-policy-controller

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.redhat.com/browse/AUTH-26

Go 1.16 update required for https://github.com/openshift/cluster-policy-controller/pull/61

### CODE REVIEW

File: ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.14
+    tag: golang-1.16
```
Comment (soltysh) [inline ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml:39]: Use `rhel-8-release-golang-1.16-openshift-4.8` instead

Comment (slaskawi) [reply ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml]: Fixed. Thanks!

**Comment by slaskawi** [comment]
@s-urbaniak @stlaz @soltysh Ready for the next round of reviews.

**Comment by stlaz** [comment]
/hold
no JIRA tickets in commits

**Comment by slaskawi** [comment]
I see this is not necessary anymore. This file has been changed in f15db75faa71727d9c84ff8e47db14d304d8d513

**Pull request closed without merging**